fair_mango/metrics/base.py

```diff
@@ -421,12 +602,38 @@ def __init__(
         self.results: dict | None = None
 
     def _compute(self) -> dict[tuple, np.ndarray[float]]:
+        """Calculate the disparity in the scores between every possible pair in
+        the provided groups.
+
+        Returns
+        -------
+        dict[tuple, np.ndarray[float]]
+            a dictionary with:
+            - keys: tuple with the pair of the sensitive groups labels.
+            - values: a numpy array with the corresponding disparity.
+        """
         metric = self.metric(self.data, **self.kwargs)
         self.targets, self.metric_results = metric()
         results = calculate_disparity(self.metric_results, "ratio")
         return results
 
     def summary(self) -> dict[str, dict[str, float | tuple | None]]:
+        """Return the fairness metric value, in other words the biggest
+        disparity found with specifying the priviliged and discriminated
+        groups.
+
+        Returns
+        -------
+        dict[str, dict[str, float | tuple | None]]
+            a dictionaty with:
```

Comment:
typo: dictionaty -> dictionary

suggestion: second time for dictionaty, might be coming from copy-paste. It might be worth to ctrl+f the typos.

Reply:
Yup, it sourced from a copy paste for sure. The issue is dealt with.